### DISCUSSION TITLE
页面缩放比输入框有小问题

### DISCUSSION THREAD

**Discussion by ioplmk** [electerm/electerm#2992] [General]
![IMG_20230919_124927.jpg](https://github.com/electerm/electerm/assets/93701868/658507e7-1e0d-4636-a6c3-b5425f0f0c33)

如果在输入框中输入缩放比 比如115% 之后会显示这种很长的浮点数。
而且还有几率会导致整个软件重启，打开的连接会全部关闭。

现在输入框中已经提供了↑↓按钮来进行加减，之前的＋－按钮显得多余了。

我觉得可不可以定一个上下限，在输完数字之后提供一个确认按钮，点击之后再生效。目前是输入3位数字后就自动生效了。

**Comment by zxdong262** [comment]
感谢反馈，将在下个版本修复
